cmd/crc/cmd/version.go

```diff
@@ -39,11 +40,11 @@ type version struct {
 	PodmanVersion    string `json:"podmanVersion"`
 }
 
-func defaultVersion() *version {
+func defaultVersion(preset crcPreset.Preset) *version {
 	return &version{
 		Version:          crcversion.GetCRCVersion(),
 		Commit:           crcversion.GetCommitSha(),
-		OpenshiftVersion: crcversion.GetBundleVersion(),
+		OpenshiftVersion: crcversion.GetBundleVersion(preset),
 		PodmanVersion:    crcversion.GetPodmanVersion(),
```

Comment:
Looks like after this PR we'll be reporting either (OpenShiftVersion, PodmanVersion) or (OKDVersion, PodmanVersion) depending on the preset? Wondering if we should always be showing all 3 versions, OpenShiftVersion, PodmanVersion, OKDVersion?

Reply:
@cfergeau Moving forward, we don't want to show any preset version as part of `crc version` command.  As of now it make sense because our crc release are bound to a specific preset bundle which will change. `crc status` should be source of truth around what preset running and which version. So as of now I don't want to change it so you are right currently it  (OpenShiftVersion, PodmanVersion) or (OKDVersion, PodmanVersion).

Reply:
I know we have plans to change this in the future. However, this is not done yet, and right now we're getting:
```
$ ./out/linux-amd64/crc version
CRC version: 2.6.0+f1c25cbc
OpenShift version: 4.10.0-0.okd-2022-06-24-212905
Podman version: 4.1.0
```
which is misleading and wrong. I don't mind if this becomes
```
$ ./out/linux-amd64/crc version
CRC version: 2.6.0+f1c25cbc
OKD version: 4.10.0-0.okd-2022-06-24-212905
Podman version: 4.1.0
```
or 
```
$ ./out/linux-amd64/crc version
CRC version: 2.6.0+f1c25cbc
OpenShift version: 4.10.22
OKD version: 4.10.0-0.okd-2022-06-24-212905
Podman version: 4.1.0
```
even though I have a preference for the latter as this means `crc version` is not going to change output depending on the preset.